website/content/packages/tailscale.md

```diff
@@ -133,6 +133,61 @@ instance to not require login every 3 months.
 [Tailscale console]: https://login.tailscale.com/ "Tailscale management console login.tailscale.com"
 {{% /notice %}}
 
+## Optional: Using gokrazy as a Tailscale subnet router
+
+To allow hosts in your Tailnet to access devices in your network other
+than your gokrazy appliance, you can set up Tailscale as a subnet router.
+
+{{< highlight json "hl_lines=12" >}}
+{
+    "Hostname": "ts",
+    "Packages": [
+        "github.com/gokrazy/serial-busybox",
+        "tailscale.com/cmd/tailscaled",
+        "tailscale.com/cmd/tailscale"
+    ],
+    "PackageConfig": {
+        "tailscale.com/cmd/tailscale": {
+            "CommandLineFlags": [
+                "up",
+                "--advertise-routes=192.168.0.0/24"
+            ]
+        }
+    }
+}
+{{< /highlight >}}
+
+Starting Tailscale version v1.63.x ([FIXME](https://github.com/tailscale/tailscale/issues/11405))
```

Comment:
It’s been a while without progress on this PR. How do you want to proceed? Do you want to split this PR into two so that we can merge the SSH part right now? Or document a small program to set the required sysctl?

Reply:
I'm hopeful Tailscale will merge a version of the PR I made as Brad Fitzpatrick suggested it'd be likely reasonable for Tailscale to do that on gokrazy. Too bad it's taking longer than I thought to get triaged.

Could we instruct how to pick the version from the PR? I think a separate program for sysctl would work as well but as I'm already using the version of tailscale from my PR, I wouldn't need it. If you think such example program serves some more general example, then that would make sense.

Reply:
> Could we instruct how to pick the version from the PR? I think a separate program for sysctl would work as well but as I'm already using the version of tailscale from my PR, I wouldn't need it. If you think such example program serves some more general example, then that would make sense.

I’d rather not. If the PR won’t get merged in its current form, users are left in the rain. Maybe it’s best to just hold off on this PR for now until the other one moves.